src/js/Component/DataList.js

```diff
@@ -127,6 +134,10 @@ Wui.DataList.prototype = $.extend(new Wui.O(), new Wui.Data(), {
                         if( me.selected && me.selected[0] && document.activeElement == me.selected[0].el[0] && me.keyActions.hasOwnProperty(evnt.keyCode) )
                             me.keyActions[evnt.keyCode].call(me);
                     });
+
+                    // columns and sorting on multiple columns
+                    me.cols = [];
```

Comment:
I wonder if they should still be named `cols` in the DataList? At this level, they're really keys to the data objects right? Do we deal with columns here?